scanner/src/test/kotlin/scanners/scancode/ScanCodeResultParserTest.kt

```diff
@@ -806,3 +508,25 @@ class ScanCodeResultParserTest : WordSpec({
         }
     }
 })
+
+private fun containLicensesExactly(vararg licenses: String): Matcher<ScanSummary?> =
+    transformingCollectionMatcher(expected = licenses.toList(), matcher = ::containExactlyInAnyOrder) { summary ->
+        summary.licenseFindings.map { it.license.toString() }.toSet()
+    }
+
+private fun containLocationsForLicenseExactly(license: String, vararg locations: TextLocation): Matcher<ScanSummary?> =
+    transformingCollectionMatcher(expected = locations.toList(), matcher = ::containExactlyInAnyOrder) { summary ->
+        summary.licenseFindings.filter { it.license.toString() == license }.map { it.location }
+    }
+
+private fun containCopyrightsExactly(vararg copyrights: Pair<String, List<TextLocation>>): Matcher<ScanSummary?> =
+    transformingCollectionMatcher(expected = copyrights.toList(), matcher = ::containExactlyInAnyOrder) { summary ->
+        summary.copyrightFindings.groupBy { it.statement }.entries
+            .map { (key, value) -> key to value.map { it.location } }
```

Comment:
Nit: I would have expected the formatting to become

    summary.copyrightFindings.groupBy { it.statement }.entries.map { (key, value) ->
        key to value.map { it.location }
    }

but fine with me 😉